spec/basic/expected_conditions_spec.js

```diff
@@ -66,6 +66,18 @@ describe('expected conditions', function() {
     expect(EC.titleIs('My AngularJS App').call()).toBe(true);
   });
 
+  it('should have urlContains', function() {
+      var baseUrlFromSpec = browser.baseUrl;
+      expect(EC.urlContains('/form').call()).toBe(true);
+      expect(EC.urlContains(baseUrlFromSpec+ 'index.html#/form').call()).toBe(true);
+  });
+
+  it('should have urlIs', function() {
+      var baseUrlFromSpec = browser.baseUrl;
+      expect(EC.urlIs('http://localhost:8081').call()).toBe(false);
```

Comment:
Nit: Instead of using this url, we should use the one provided from the environment variables:
```
var env = require('../environment');
expect(EC.urlIs(env.baseUrl).call()).toBe(false);
```